docs/UserGuide.md

```diff
@@ -80,6 +80,7 @@ The `Chart Panel` (an example is shown above) contains _Ramp Charts_ and _Contri
 
 The `Code Panel` allows users to see the code attributed to a specific author. Click on the name of the author in the `Chart Panel` to display the `Code Panel` on the right.
 * The Code Panel shows the files that contain author's contributions, sorted by the number of lines written.
+* To view code from certain language, click on the checkboxes to choose.
```

Comment:
We are filtering by `file extensions` and not quite of `language`/`syntax`.

Let's also make it more concise.

Quick Suggestion:
`Select or deselect the checkboxes to include or exclude files of selected formats.`

Also, we may need to standardize usage of file `formats` or `extensions`.